src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -140,36 +140,44 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
       .subscribe(v => {
         const event: RecordingNotificationEvent = v.message;
         notifications.info('Recording Created', `${event.recording} created in target: ${event.target}`);
-        refreshRecordingList();
+        React.useCallback(() => {
+          refreshRecordingList();
+        }, [refreshRecordingList]);
```

Comment:
Does `refreshRecordingList` actually get called when this is done? From my understanding of the `useCallback` hook, this returns a memoized wrapper of the provided function (which is in this case actually a lambda/fat-arrow-function), but I'm not sure that it will actually cause the function (memoized version or original) to be invoked.